### PR TITLE
Fix #119, add issue and pull request templates

### LINKED ISSUE #119: Add issue and PR templates similar to those used in CFS
BP and BPLib development needs to follow NPR 7150, and part of the documentation requirements are fulfilled by tracking who is requesting changes and who is implementing them, along with why it is needed.

CFS has templates for issues and PRs that aim to capture this information geared toward fulfilling those requirements.  BP and BPLib should be able to use the same templates.

### PR DESCRIPTION
**Pull request by jphickey**
These are the same issue and PR templates as other CFS apps use.

Fixes #119

### CODE REVIEW

**Review by dzbaker — APPROVED** [review]
Looks good as a start other than the CLAs (they point to cFE).

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+**Third party code**
+If included, identify any third party code and provide text file of license
+
+**Contributor Info - All information REQUIRED for consideration of pull request**
+Full name and company/organization/center of all contributors ("Personal" if individual work)
+- If NASA Civil Servant Employee or GSFC Contractor on SES II
+  - Address/email/phone and contract/task information (if applicable) must be on file
+- Else if Company
+  - **HAND SIGNED** Company CLA must be on file (once per release): [Company CLA](https://github.com/nasa/cFE/blob/master/docs/GSC_18128_Corp_CLA_form_1219.pdf)
```
[the remark this answers is not in the document]
Comment (jphickey) [reply .github/pull_request_template.md:32]: Good point, do we have a CLA for BPLib?

The other CFS apps reference a CONTRIBUTING.md document that we also don't have for BPLib.

So - should we hold off on this until we have those documents, or just use the CFE ones for now?

I also noticed it says `SES II` where I should probably update that to say `SES III` (same with CFE)

[the remark this answers is not in the document]
Comment (dzbaker) [reply .github/pull_request_template.md:32]: Wouldn't these be the CLAs? https://github.com/nasa/bplib/tree/main/doc

[the remark this answers is not in the document]
Comment (jphickey) [reply .github/pull_request_template.md:32]: I forgot those were in there ... but they are outdated, I think - that was for the NOSA software under the previous NTR.  The current NTR is officially: `GSC-18,587-1 and identified as “The Bundle Protocol Core Flight System Application (BP) v6.5”`

Perhaps its just as simple as changing the name, but I am not sure.

**Comment by jphickey** [comment]
Adding the read tag back, as this might need some discussion about what to do next

**Comment by jphickey** [comment]
Discussed in 2022-10-26 CCB, update to point to existing CLA (even if outdated) and CLA will be updated in future

**Comment by jphickey** [comment]
PR has been updated as requested.  @dzbaker - can you re-review and confirm?

**Review by dzbaker — APPROVED** [review]

**Comment by jphickey** [comment]
Received confirmation from PDL that this can now be approved, label switched

**Pull request merged**